src/main/java/make_comment/use_case/MakeCommentInteractor.java

```diff
@@ -0,0 +1,104 @@
+package make_comment.use_case;
+
+
+import entities.Comment;
+import entities.CurrentUser;
+
+
+import java.util.*;
+
+/**
+ * The MakeCommentInteractor in the use case layer implements the MakeCommentInputBoundary interface.
+ * It takes in information from the outer layers of the architecture to run the use case by working directly
+ * with the entities and ultimately send information back out into the UI.
+ */
+public class MakeCommentInteractor implements MakeCommentInputBoundary {
+    private final MakeCommentDSGateway dataAccess;
+    private final MakeCommentOutputBoundary presenter;
+
+    private final CommentFactory commentFactory = new CommentFactory();
+
+    public MakeCommentInteractor(MakeCommentDSGateway dataAccess, MakeCommentOutputBoundary presenter) {
```

Comment:
can we pass the commentfactory in through the constructor? its like a convention we're doing elsewhere